plugins/source/aws/client/testing.go

```diff
@@ -90,7 +90,7 @@ func AwsCreateMockClient(t *testing.T, ctrl *gomock.Controller, builder func(*te
 
 func validateTagStructure(t *testing.T, tables schema.Tables) {
 	for _, table := range tables.FlattenTables() {
-		t.Run(table.Name, func(t *testing.T) {
+		t.Run("validate tags", func(t *testing.T) {
```

Comment:
```suggestion
		t.Run("validate tags " + table.Name, func(t *testing.T) {
```
maybe?

Reply:
I was going to use `:` and so on as separator but in the end they all get converted to `_` anyway

Reply:
Addressed in 7aaa291ecb0ba282223bee6b8a80066c5279a7b4